go.mod

```diff
@@ -24,6 +23,7 @@ require (
 	github.com/gin-gonic/gin v1.7.7
 	github.com/gizak/termui/v3 v3.1.0
 	github.com/gogo/protobuf v1.3.2
+	github.com/google/go-cmp v0.5.6 // indirect
```

Comment:
might run `go mod tidy` again ? also, you could remove `go.sum` and run `go mod tidy` again, maybe these `// indirect` will disappear